src/test_deque.py

```diff
@@ -0,0 +1,174 @@
+"""Test methods of the deque data structure."""
+
+import pytest
+
+
+@pytest.fixture
+def sample_deque():
+    """Create sample deque."""
+    from deque import Deque
+    new_queue = Deque([1, 2, 3, 4, 5])
+    return new_queue
+
+
+@pytest.fixture
+def single_deque():
+    """Create one length deque."""
+    from deque import Deque
+    one_queue = Deque(["one"])
+    return one_queue
+
+
+@pytest.fixture
+def empty_deque():
+    """Create empty deque."""
+    from deque import Deque
+    empty_queue = Deque()
+    return empty_queue
+
+
+def test_deque_empty_init(empty_deque):
+    """Test for empty deque init."""
+    assert empty_deque.dll.head_node is None
+    assert empty_deque.dll.tail_node is None
+
+
+def test_deque_one_init(single_deque):
+    """Test for one item deque init."""
+    assert single_deque.dll.head_node.contents == "one"
+    assert single_deque.dll.tail_node.contents == "one"
+
+
+def test_deque_init(sample_deque):
+    """Test for new deque init."""
+    assert sample_deque.dll.head_node.contents == 5
+    assert sample_deque.dll.tail_node.contents == 1
```

Comment:
Split this into two tests.  One that confirms head_node and one that confirms tail_node.  You could also make a test to verify size is 0.